packages/vue/src/vue-component-lib/utils.ts

```diff
@@ -49,7 +49,7 @@ const getElementClasses = (ref: Ref<HTMLElement | undefined>, componentClasses:
 * @prop externalModelUpdateEvent - The external event to fire from your Vue component when modelUpdateEvent fires. This is used for ensuring that v-model references have been
 * correctly updated when a user's event callback fires.
 */
-export const defineContainer = <Props, VModelType=string|number|boolean>(
+export const defineContainer = <Props, VModelType = string | number | boolean>(
```

Comment:
Any idea why this is happening?

Reply:
Probably just my editor prettifying it when I reverted the change, I'll revert that too.